.devcontainer.json

```diff
@@ -13,5 +13,9 @@
 			]
 		}
 	},
-	"remoteUser": "simbricks"
-}
+	"remoteUser": "simbricks",
+	"remoteEnv": {
+		"PATH": "/home/simbricks/.local/bin:${containerEnv:PATH}"
+	},
+	"onCreateCommand": "pip install -r requirements.txt"
```

Comment:
Why not just install these in the Dockerfile? Seems a bit silly to install locally for everyone opening the container if we can avoid it. Or am I missing something?

Reply:
I agree. There's no particular reason not to do this in the Dockerfile directly.